src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -203,23 +279,119 @@ static ucs_status_t uct_tcp_sockcm_ep_server_invoke_conn_req_cb(uct_tcp_sockcm_e
     return UCS_OK;
 }
 
+void uct_tcp_sockcm_ep_client_connect_cb(uct_tcp_sockcm_ep_t *cep,
+                                         uct_cm_remote_data_t *remote_data,
+                                         ucs_status_t status)
+{
+    cep->state |= UCT_TCP_SOCKCM_EP_CONN_NOTIFY_CB_INVOKED;
```

Comment:
i wanted a common flag for client and server -
for client - **CONN**_NOTIFY_CB_INVOKED

for server - CONN_**NOTIFY**_CB_INVOKED

Reply:
i think it's confusing using same terminology for multiple cases, notify is only when the client tells the server it has connected, calling the "notify" UCT api function

Reply:
this flag is used on each side to indicate the connect_callback was invoked.
so on the client it's set before sending the notify to the server.
since on the server side, it's called notify_cb and not connect_cb, i added the NOTIFY to the flag.

should we have a separate flag for each side to indicate that its connect/notify_cb was invoked?

Reply:
yes, there should be separate flag for each type of callback, easier to debug and understand, no perf impact